modules/appeals_api/spec/fixtures/valid_200996_v2_extra.json

```diff
@@ -69,7 +70,8 @@
         "type": "contestableIssue",
         "attributes": {
           "issue": "left ear",
-          "decisionDate": "1900-01-04"
+          "decisionDate": "1900-01-04",
+          "socDate": "2020-04-30"
```

Comment:
Let's keep each of these SOC dates unique so that we can verify they're not just accidentally copying from another issue by some crazy fluke.

Reply:
Ah. Good call.